File: Problems/Count-Odd-Numbers-in-an-Interval-Range/workArea/countOddNumbersInAnInterval.c
int countOdds(int low, int high){
    /*
    int count = 0;
    for(int i = low; i <= high; i++) {
        if(i%2 != 0) {
            count++;
        } 
        else {
            continue;
        }
    }
    
    return count;*/
    return ((high+1) >> 1) - (low >> 1);
}

File: Problems/Count-Odd-Numbers-in-an-Interval-Range/workArea/countOddNumbersInAnInterval.py
class Solution:
    def countOdds(self, low: int, high: int) -> int:
        '''count = 0
        for i in range(low, high+1):
            if i%2 != 0:
                count += 1
            else:
                continue
            
        return count'''
        
        return (high+1)//2 - (low)//2